packages/storybook-test/src/I18n.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Decorator } from '@storybook/react';
+import { useEffect } from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { i18n } from './i18n';
+
+i18n.on('languageChanged', function changeDocumentDirection(locale) {
+  document.dir = i18n.dir(locale);
+  document.documentElement.lang = locale;
+});
+
+export const I18n: Decorator = (Story, context) => {
+  const { locale } = context.globals;
+
+  useEffect(() => {
+    i18n.changeLanguage(locale);
+
+    return () => {
+      document.dir = 'ltr';
```

Comment:
`document.dir`, bizar en geinig dat het kan!

Maar waarom zou je niet ook `documentElement.dir` instellen? Logisch en gebruikelijk om het op 't `<html>` element te doen toch?

Reply:
Dit doet sowieso nog niet helemaal wat ik zou willen:

<img width="1104" alt="screenshot waarin de hele pagina right-to-left is in plaats van een enkele story" src="https://github.com/user-attachments/assets/01b83e6c-cd90-4f36-a8c1-2bdf74bc92bb">